convert-hf-to-gguf.py

```diff
@@ -253,7 +253,7 @@ def _get_model_architecture(self) -> gguf.MODEL_ARCH:
             return gguf.MODEL_ARCH.REFACT
         if arch == "PersimmonForCausalLM":
             return gguf.MODEL_ARCH.PERSIMMON
-        if arch in ("StableLMEpochForCausalLM", "LlavaStableLMEpochForCausalLM"):
+        if arch in ("StableLmForCausalLM", "LlavaStableLMEpochForCausalLM"):
```

Comment:
Likewise, for compatibility with finetunes which won't necessarily update their `config.json`
```suggestion
        if arch in ("StableLmForCausalLM", "StableLMEpochForCausalLM", "LlavaStableLMEpochForCausalLM"):
```